se/browser.py

```diff
@@ -43,9 +44,13 @@ def initialize_selenium_firefox_webdriver() -> webdriver.firefox.webdriver.WebDr
 	profile.set_preference("browser.http.use-cache", False)
 	profile.set_preference("layout.css.devPixelsPerPx", "2.0")
 
+	if platform.system() == "Linux":
```

Comment:
I think this should be the other way around. Instead of testing for a platform, we should test for the presence of `geckdriver` with `shutil.which()`. If it exists we can use the system `geckodriver`; otherwise if we're on Linux then use our packaged one, and finally if we're not on Linux, fatal error with instructions to install `geckodriver` into `$PATH`

Reply:
Although maybe we should just include a binary of `geckodriver` for each 64-bit Win/Mac/Linux platform and test the platform to use the appropriate one that we package? They're not big: https://github.com/mozilla/geckodriver/releases

Reply:
I’m not sure why you vendored it in the first place?

Either way, if we include it we’ll need mac Intel and ARM, Linux Intel (and potentially ARM?) and Windows Intel (and at a push ARM, though I’m not sure anyone’s devving on ARM there). Personally I think we’re better off packaging the Intel Linux build we use on the server, and letting other people install manually.

Reply:
Because on Linux it's not usually in the package manager so it's more annoying to install, and it's a small binary that's only updated semi-yearly so it's easy enough to package. Also, that way we have consistent results across toolset installs and we can't potentially be confused by someone's differing driver version.

Now that I think more about it, let's just package it for 64 bit Intel on all platforms, plus 64 bit ARM for Mac only. Is it possible to test for Intel vs ARM with `platform`?

Reply:
I lied, it was `platform.machine()` in the end.

Reply:
Ah, hang on, that’s not working. One sec.

Reply:
This is now building _A Tangled Tale_ fine for me locally.